lib/widgets/poll.dart

```diff
@@ -69,46 +86,70 @@ class _PollWidgetState extends State<PollWidget> {
           store.users[userId]?.fullName ?? zulipLocalizations.unknownUserName)
         .join(', ');
 
-      return Padding(
-        padding: const EdgeInsets.only(bottom: 5),
-        child: Row(
-          spacing: 5,
-          crossAxisAlignment: CrossAxisAlignment.baseline,
-          textBaseline: localizedTextBaseline(context),
-          children: [
-            ConstrainedBox(
-              constraints: const BoxConstraints(minWidth: 25),
-              child: Container(
-                height: 25,
-                padding: const EdgeInsets.symmetric(horizontal: 4),
-                decoration: BoxDecoration(
-                  color: theme.colorPollVoteCountBackground,
-                  border: Border.all(color: theme.colorPollVoteCountBorder),
-                  borderRadius: BorderRadius.circular(3)),
-                child: Center(
-                  child: Text(option.voters.length.toString(),
-                    textAlign: TextAlign.center,
-                    style: textStyleBold.copyWith(
-                      color: theme.colorPollVoteCountText, fontSize: 13))))),
-            Expanded(
+      return Row(
+        crossAxisAlignment: CrossAxisAlignment.baseline,
+        textBaseline: localizedTextBaseline(context),
+        children: [
+          GestureDetector(
+            // TODO: Implement feedback when the user taps the button
+            onTap: () => _toggleVote(option),
+            behavior: HitTestBehavior.translucent,
+            child: ConstrainedBox(
+              constraints: const BoxConstraints(
+                minWidth: 39 + 5, minHeight: 39 + verticalPadding * 2),
+              child: Padding(
+                // For accessibility, the touch target is padded to be larger
+                // than the vote count box.  Still, we avoid padding at the
+                // start because we want to align all the poll options to the
+                // surrounding messages.
+                padding: const EdgeInsetsDirectional.only(
+                  end: 5, top: verticalPadding, bottom: verticalPadding),
```

Comment:
nit:

    poll [nfc]: Move vote count paddings to a box

I'm not sure what "to a box" means — `Padding` is always a box, right?

Could say:

> poll [nfc]: Move vote count paddings inside children of Row

and I think that accurately conveys what this commit is doing.

Reply:
other nit in same commit message:

    specified on the outer (now removed) Padding.  The Container is
    expanded because it now also contains the paddings.

I think you mean the ConstrainedBox, not Container.